### DISCUSSION TITLE
Is there a way to interact with the shadow thread directly?

### DISCUSSION THREAD

**Discussion by malikKartik** [react-native-community/discussions-and-proposals#593] [Ideas]
Hey everyone, I want to calculate the layout of certain elements using yoga which RN already does. 

### Why I can't use  RN's own flex-box?
I am creating a few elements on react-native-skia canvas and I want to use a layout engine for that I can't use RN flexbox directly there as it is not supported.

Hence I am looking for a way to directly interact with the yoga functionality that runs on the shadow thread is there a way to do that?

**Comment by kelset** [comment]
👋 there - as stated in the README and the issue template, this repo is for long-form discussions. Yours seems more a question for [StackOverflow instead](https://stackoverflow.com/questions/tagged/react-native).
